### PR TITLE
feat: Add windows server 2022 autounattend config map

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
Add windows server 2022 autounattend config map
This commit adds new configmap for windows server 2022 and renames
windows11-installer pipeline to windows-efi-installer pipeline.
With this change user can install other windows which requires efi +
secure boot with single pipeline.

**Release note**:
```
Add windows server 2022 autounattend config map
```

### CODE REVIEW

**Review by codingben — APPROVED** [review]
/lgtm

File: data/tekton-pipelines/okd/windows-efi-installer.yaml [hunk 1]
```diff
@@ -257,8 +257,8 @@ spec:
           value: $(tasks.copy-template.results.name)
         - name: templateAnnotations
           value:
-            - "openshift.io/display-name: Microsoft Windows 11 Installer VM"
-            - "description: Template for installing Microsoft Windows 11."
+            - 'openshift.io/display-name: Microsoft Windows Installer VM'
```
Comment (0xFelix) [inline data/tekton-pipelines/okd/windows-efi-installer.yaml:260]: nit: single/double quotes

Otherwise LGTM!

/lgtm

**Comment by ksimon1** [comment]
/hold
Currently investigating missing drivers in virtio image for win 2022

**Pull request merged**